### PR TITLE
Clean dependency

### PR DESCRIPTION
**Pull request by Sommerbo**
Removed "Using UnityEditor;" line and cleaned up the code. Functionality remains the same but this doesn't block building the project to a stand-alone program.
This supersedes #456

### CODE REVIEW

File: Assets/Scripts/UI/DialogBox/FileSaveLoad/DialogBoxLoadGame.cs [hunk 1]
```diff
@@ -115,11 +115,12 @@ public void DeleteFile()
 		}
 
 		CloseSureDialog();
-		FileUtil.DeleteFileOrDirectory(filePath);
+		File.Delete(filePath);
```
Comment (Mizipzor) [inline Assets/Scripts/UI/DialogBox/FileSaveLoad/DialogBoxLoadGame.cs]: Spaces, not tabs. :)

File: Assets/Scripts/UI/DialogBox/FileSaveLoad/DialogBoxLoadGame.cs [hunk 2]
```diff
@@ -115,11 +115,12 @@ public void DeleteFile()
...
 		CloseDialog();
 		ShowDialog();
 	}
 
+	
 	public void CloseSureDialog()
```
Comment (Mizipzor) [inline Assets/Scripts/UI/DialogBox/FileSaveLoad/DialogBoxLoadGame.cs]: Trailing whitespace is never useful anywhere.

**Comment by Sommerbo** [comment]
@Mizipzor I saw that you merged my earlier PR with this one, could I have done that?

**Comment by Mizipzor** [comment]
@Sommerbo Hm, Im not sure what you mean. What earlier PR did I merge?

**Comment by Sommerbo** [comment]
@Mizipzor Sorry, I read it wrong. It was referenced by me no less.

**Comment by vogonistic** [comment]
Merged!

**Pull request merged**